pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -424,13 +424,13 @@ func (m *ManagerImpl) Register(ctx context.Context, r *pluginapi.RegisterRequest
 	}
 	if !versionCompatible {
 		errorString := fmt.Sprintf(errUnsupportedVersion, r.Version, pluginapi.SupportedVersions)
-		klog.Infof("Bad registration request from device plugin with resource name %q: %s", r.ResourceName, errorString)
+		klog.InfoS("Bad registration request from device plugin with resource.", "resource", r.ResourceName, "error", errorString)
 		return &pluginapi.Empty{}, fmt.Errorf(errorString)
```

Comment:
Let's pass the error from `fmt.Errorf(errorString)` directly to InfoS under `err` key

Reply:
Is it required for all ocurrences of err on InfoS?